samples/sampler/stories/default/PopupTabLayout.js

```diff
@@ -89,11 +101,11 @@ export const _PopupTabLayout = () => {
 					onTabClick={action('onTabClick')}
 					title="Display"
 				>
-					<TabPanels index={indexDisplay} onBack={handleDisplayPrev} onClose={handleClose}>
+					<TabPanels index={indexDisplay} onBack={handleDisplayPrev}>
```

Comment:
Could you leave the reason why the onClose is removed on description?

Reply:
Oh.. it should've been removed from #888 since we are now don't close the popup on the TabPanels.